www/gatsby-config.js

```diff
@@ -75,6 +78,67 @@ module.exports = {
         trackingId: 'UA-41298772-1',
       },
     },
+    {
+      resolve: 'gatsby-plugin-feed',
+      options: {
+        query: `
+         {
+          site {
+            siteMetadata {
+              rssFeedTitle
+              title: rssFeedTitle
+              rssFeedDescription
+              description: rssFeedDescription
```

Comment:
Why pass `rssFeedTitle` and `rssFeedDescription`? I think it should be sufficient to pass these via the re-named `title` and `description` keys only.

Reply:
Correct; apparently the only field that needs to be duplicate is the siteURL field, otherwise the links in the RSS feed don't work. My latest commit should reflect those changes.